source/includes/inkling-reference/_lessons.html.md

```diff
@@ -137,8 +137,6 @@ lessonStatement ::=
     testClause?
 ```
 
-Select the Syntax tab to see the lesson syntax. The syntax for lesson and its subordinate
```

Comment:
In my version there is no syntax vs code tab anymore. But if you don't want the change that is fine.